main.js

```diff
@@ -0,0 +1,123 @@
+const apiKey = "6d29bbab";
+
+document.getElementById("form-submit").addEventListener("submit", getIp);
+let page = 1;
+let searchWord = document.getElementById("searchWord");
+function getIp(e) {
+  e.preventDefault();
+  let iVal = searchWord.value;
+  search(iVal, page);
+}
+
+const renderMovieNotFound = (error = "Movie not found!") => {
+  const movieNotFoundContainer = document.createElement("div");
+  movieNotFoundContainer.innerHTML = confirm(error);
```

Comment:
You need to clean all previous movies after each click on "Search"
And do not use `confirm`. It will return undefined into `movieNotFoundContainer.innerHTML`

https://user-images.githubusercontent.com/34886027/203255633-99ff1dd5-a4b5-424c-85fd-2cdab4090da4.mov